### PR TITLE
seed rembadger vault w/ 600k $badger from treasury

### PR DESCRIPTION
**Pull request by jorijnsmit**
see #12 for details

### CODE REVIEW

File: scripts/issue/12/seed_rembadger.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    mantissa = TO_SEED * 10 ** badger.decimals()
+    assert badger.balanceOf(safe) >= mantissa
+
+    # get current balance of rembadger vault
+    bal_before = badger.balanceOf(rembadger)
+    pps_before = rembadger.getPricePerFullShare()
+
+    # transfer
+    badger.approve(rembadger, mantissa)
```
Comment (petrovska-petro) [inline scripts/issue/12/seed_rembadger.py:34]: i think here the approve is not needed

Comment (GalloDaSballo) [reply scripts/issue/12/seed_rembadger.py:34]: Agreed

Comment (jorijnsmit) [reply scripts/issue/12/seed_rembadger.py:34]: ah yea thats still a relic from when it wasnt `transfer` but `addWant`.

**Review by petrovska-petro — APPROVED** [review]

**Review by GalloDaSballo — APPROVED** [review]
LGTM

Will seed the pool with 600k Badger and does a couple checks
Screenshot 2022-01-26 at 12 58 25

Screenshot 2022-01-26 at 12 58 30

Let's goo!

**Pull request merged**